examples/example_host.cpp

```diff
@@ -52,15 +52,19 @@ static bool account_exists(evmc_context* context, const evmc_address* address)
     return host->accounts.find(*address) != host->accounts.end();
 }
 
-static void get_storage(evmc_uint256be* result,
+static bool get_storage(evmc_uint256be* result,
                         evmc_context* context,
                         const evmc_address* address,
                         const evmc_uint256be* key)
 {
-    (void)result;
-    (void)context;
-    (void)address;
-    (void)key;
+    example_host_context* host = static_cast<example_host_context*>(context);
+    auto it = host->accounts.find(*address);
+    if (it != host->accounts.end())
+    {
+        *result = it->second.storage[*key];
```

Comment:
What happens when the key is not found?

Reply:
It magically inserts the null hash to the map at this position and returns it.

Reply:
Ah right because it is the index and not `.at`. Damn C++, damn.

Reply:
I think this is mostly to allow insertion in form of `m["hello"] = "world"`.